webrtc.html

```diff
@@ -11167,8 +11167,21 @@ <h2>Dictionary <a class="idlType">RTCStats</a> Members</h2>
               <code><a>RTCStats</a></code> objects, extracted from two
               different <code><a>RTCStatsReport</a></code> objects, MUST have
               the same id if they were produced by inspecting the same
-              underlying object. User agents are free to pick any format for
+              underlying object.</p>
+              <p>Stats ids MUST be alphanumerical. Stats ids MUST NOT be
```

Comment:
Remove "alphanumerical" from this PR and sort out allowed characters and lengths in a separate issue/PR.